### PR TITLE
[feat]: Updated the carousel with latest proposed design

### PR DESCRIPTION
**Pull request by red-game-dev**

### CODE REVIEW

**Review by BrettCleary — APPROVED** [review]
lgtm

File: src/components/QuestsBanner/QuestsBanner.module.scss [hunk 1]
```diff
@@ -14,14 +14,16 @@ $bannerCarouselWidth: 747px;
...
-    align-items: flex-start;
     flex-direction: row;
-    gap: 60px;
-    height: 270px;
-    max-height: 270px;
-    padding: 0 80px;
+    gap: 40px;
+    height: 250px;
+    max-height: 250px;
+    padding: var(--space-xl) 120px;
```
Comment (BrettCleary) [inline src/components/QuestsBanner/QuestsBanner.module.scss:26]: you may want this to be space-xl-fixed

File: src/components/QuestsBanner/QuestsBanner.module.scss [hunk 2]
```diff
@@ -14,14 +14,16 @@ $bannerCarouselWidth: 747px;
   position: relative;
   border-radius: var(--space-md);
   background: var(--color-neutral-700);
+  flex-shrink: 0;
+  align-self: stretch;
+  padding: var(--space-2lg);
```
Comment (BrettCleary) [inline src/components/QuestsBanner/QuestsBanner.module.scss:19]: you may want this to be space-2lg-fixed

File: src/components/QuestsBanner/QuestsBanner.module.scss (hunk 1 above)
Comment (red-game-dev) [reply src/components/QuestsBanner/QuestsBanner.module.scss:26]: gonna have a look, cheers 🙌

**Pull request merged**